contributions.json

```diff
@@ -8,5 +8,10 @@
     "contribution-title": "Add Scroll to ethereum-multicall",
     "project-repo": "https://github.com/joshstevens19/ethereum-multicall/",
     "contributors": ["omahs"]
+  },
+  {
+    "contribution-title": "Update README",
+    "project-repo": "https://github.com/juve4ever/Scroll",
```

Comment:
Can you please set the repo to the original one?
https://github.com/scroll-tech/contribute-to-scroll

Reply:
what?

Reply:
try to integrate Scroll into JUMPER.EXCHANGE